packages/cli/src/commands/apps/oauth/generate.ts

```diff
@@ -36,7 +36,21 @@ export default class AppOauthGenerateCommand extends APICommand<typeof AppOauthG
 		const config: InputAndOutputItemConfig<GenerateAppOAuthResponse> = {
 			tableFieldDefinitions,
 		}
+		const getInputFromUser = async (): Promise<GenerateAppOAuthRequest> => {
+			const oauth = await this.client.apps.getOauth(appId)
+			const startingRequest: GenerateAppOAuthRequest = {
+				clientName: oauth.clientName,
+				scope: oauth.scope,
+			}
+			const inputDef = objectDef('Generate Request', {
+				clientName: stringDef('Client Name'),
+				scope: oauthAppScopeDef,
+			})
+
+			return updateFromUserInput(this, inputDef, startingRequest, { dryRun: false })
```

Comment:
The user has the ability to set or not set the `--dry-run` flag with this command, so we should pass that instead.